moonraker/components/history.py

```diff
@@ -170,7 +171,20 @@ async def _handle_jobs_list(self,
 
     async def _handle_job_totals(self,
                                  web_request: WebRequest
-                                 ) -> Dict[str, Dict[str, float]]:
+                                 ) -> Dict[str, Union[bool, Dict[str, float]]]:
+        action = web_request.get_action()
+        if action == "DELETE":
+            for x in self.job_totals:
+                if isinstance(self.job_totals[x], float):
+                    self.job_totals[x] = float(0.)
+                else:
+                    self.job_totals[x] = int(0)
+            database: DBComp = self.server.lookup_component("database")
+            database.insert_item(
+                "moonraker", "history.job_totals", self.job_totals)
+            return {"success": True}
```

Comment:
I would just use the same return value as the `GET` method here, essentially just the empty job totals.